### ISSUE TITLE
Flymake Error Underline Issue

### ISSUE THREAD

**Issue by kborling** [idlip/haki#6]
Hey there. Loving the theme!

When using flymake, I noticed the error lines don't display underneath the symbol. Below I've included screenshots of what it looks like using Haki vs. Modus Vivendi theme.

Modus Vivendi:
<img width="462" alt="Screenshot 2023-11-01 at 20 00 10" src="https://github.com/idlip/haki/assets/4042187/eb3bc9fc-c3ae-4a68-8b32-3c2ca2762188">

Haki:
<img width="462" alt="Screenshot 2023-11-01 at 20 01 22" src="https://github.com/idlip/haki/assets/4042187/df959488-dedb-456e-99b8-09a517aaf869">

**Comment by idlip** [comment]
Thank you for liking this theme.

I got the point, there was no face that was controlling flymake fringe indicator, but some variables 
```
flymake-error-bitmap
flymake-note-bitmap
flymake-warning-bitmap
```
control them.

So evaluating ```(customize-set-variable `flymake-error-bitmap '(flymake-double-exclamation-mark flymake-error-echo))``` will give background for flymake indicator (fringe).
In fact that is how modus-themes also set variables to get that.

Haki theme is not much matured or customizable like modus, but hopefully I will try to find a way to add more goodies.

I moved to flycheck-mode, so did not notice this as such, in flycheck it appears as you asked.
![haki-flycheck](https://github.com/idlip/haki/assets/117019901/a8c6623e-9773-49af-b461-13c15b969dc9)


I will try to add something regarding this, and in the mean you can customize those `flymake-*-bitmap` variables with faces to your liking.

Thank you for reporting it and your understanding!

**Comment by kborling** [comment]
I tried what you suggested, but it didn't work for me. Since it was working for you, I thought I would try installing the flycheck package to see if it works. Once I installed flycheck, the error highlighting worked as expected:
<img width="461" alt="Screenshot 2023-11-03 at 09 48 26" src="https://github.com/idlip/haki/assets/4042187/ac15cb0c-89d7-4baf-a9f9-0899e50a824c">

I checked how you defined the flymake faces and see these inherit from flycheck. This doesn't seem to work if the flycheck package isn't installed. This would probably work if you swapped these around and inherited flycheck from flymake, since flymake is built-in. Otherwise, redefining the face instead of inheriting would most likely fix the issue as well.

```elisp
`(flymake-error                                ((,class :inherit flycheck-error)))
`(flymake-error-echo                           ((,class :inherit error)))
`(flymake-note                                 ((,class :inherit flycheck-info)))
`(flymake-note-echo                            ((,class :inherit success)))
`(flymake-warning                              ((,class :inherit flycheck-warning)))
```
